ccm_web/client/src/App.tsx

```diff
@@ -114,6 +94,24 @@ function App (): JSX.Element {
         <Route path="/*" element={<NotFound />}/>
       </Routes>
     </Layout>
+    <HelmetProvider>
+      <Helmet>
+        <script type="text/javascript">
+          {`window.umConsentManager = {
+              googleAnalyticsID: ${globals?.googleAnalyticsId ?`'${globals?.googleAnalyticsId}'`: false},
+              googleAnalyticsCustom: {
+                  streamConfig: {
+                      cookie_flags: "SameSite=None; Secure"
+                  }
+              },
+              externalLinkBlank: true
+            }
+        `}  
+        </script>
+        <script async src="https://umich.edu/apis/umconsentmanager/consentmanager.js"/>
+      </Helmet>
+    </HelmetProvider>
+    </>
```

Comment:
In My Local CCM instance doesn't run from Loophole, I see LTI launch error and 403 errors

If I switch to Ngrok, I see LTI launches happen but Cookie banner won't launch from chrome, but it does from Edge. 

 the cookie behavior seems to be
 1.  `um_privacy_consent` usually appears before banner appears, 
 2. once User accept the cookie I see `um_privacy_conent_1222` being 
 
 So typically you would atleast see 2 cookies if user accepts
 
 From Edge I Cookies are not set, not sure this is because I am testing from Ngrok and not Loophole. I am having some difficulty with Proxy services.

Reply:
I will debug more on this tomorrow.